src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,11 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+13.0.0b10
```

Comment:
Please update the major version if you plan to make a breaking change. (e.g., 14.0.0b1)